sdks/python/apache_beam/runners/worker/sdk_worker.py

```diff
@@ -464,6 +469,7 @@ def __init__(self, credentials=None):
     self._lock = threading.Lock()
     self._throwing_state_handler = ThrowingStateHandler()
     self._credentials = credentials
+    self.state_cache = StateCache(100)
```

Comment:
do you want have this as a class variable so it can be shared across different instances of this class?